src/coresyftools/coresyftools/tests/test_tester.py

```diff
@@ -1,12 +1,28 @@
 from unittest import TestCase
 
 from ..tool_tester import ToolTester, NonZeroReturnCode, NonEmptyStderr, NoOutputFile, EmptyOutputFile
+import os
 
 
 class TestTester(TestCase):
 
+    def test_download_input(self):
+        tester = ToolTester('coresyftools/tests/dummy_tool',
```

Comment:
You should not hardcode this. You should get the `abspath` of the test file (os.path.absptah(__file__)) and specify dirs based on this path.